vector/src/main/java/im/vector/app/features/html/FontTagHandler.kt

```diff
@@ -31,7 +32,13 @@ class FontTagHandler : SimpleTagHandler() {
 
     override fun getSpans(configuration: MarkwonConfiguration, renderProps: RenderProps, tag: HtmlTag): Any? {
         val colorString = tag.attributes()["color"]?.let { parseColor(it) } ?: Color.BLACK
-        return ForegroundColorSpan(colorString)
+        val bgColor = tag.attributes()["data-mx-bg-color"]
+        if(bgColor == null) {
+            return ForegroundColorSpan(colorString)
+        }
+        else {
+            return arrayOf(ForegroundColorSpan(colorString), BackgroundColorSpan(bgColor.let { parseColor(it) }))
+        }
```

Comment:
`parseColor` will fallback to BLACK in case of error, so it can lead to text rendered in black, on a black background.
Maybe add a parameter to `pasreColor` with the color to use in case of error. I will use TRANSPARENT for default bg color.

Anyway, using BLACK color for default text color is not good too, because text color depends on the current theme. But this is another story, and not related to this PR

Reply:
Done :-)